core/arch/arm/plat-ti/platform_config.h

```diff
@@ -153,9 +153,28 @@
 
 #endif /* CFG_WITH_PAGER */
 
+#if defined(CFG_SECURE_DATA_PATH)
+
+#if !defined(CFG_TEE_SDP_MEM_SIZE)
+#define CFG_TEE_SDP_MEM_SIZE    0x00400000
+#endif
+
+/* default locate SDP memory at the end of TZDRAM */
+#if !defined(CFG_TEE_SDP_MEM_BASE)
+#define CFG_TEE_SDP_MEM_BASE	(TZDRAM_BASE + \
+				TZDRAM_SIZE - \
+				CFG_TEE_SDP_MEM_SIZE)
+#endif
+
+#else /* CFG_SECURE_DATA_PATH */
+#define CFG_TEE_SDP_MEM_SIZE    0
+#endif /* CFG_SECURE_DATA_PATH */
```

Comment:
Ideally, I'd like all `CFG_` variables to be configurable from makefiles or the command line, not set in `.h` files (for simplicity and flexibility). I know this is not currently not done consistently, but how about doing just:
```C
#if defined(CFG_SECURE_DATA_PATH)
#define CFG_TEE_SDP_MEM_BASE	(TZDRAM_BASE + \
				TZDRAM_SIZE - \
				CFG_TEE_SDP_MEM_SIZE)
#endif
```
(because this computation it cannot be done practically in the makefiles)
And set `CFG_TEE_SDP_MEM_SIZE ?= 0x00400000` in `plat-ti/conf.mk`.

?

Reply:
Works for me, updated.